include/effects/matrix/PatternSerendipity.h

```diff
@@ -94,7 +94,7 @@ class PatternSerendipity : public LEDStripEffect
     }
 
 public:
-    PatternSerendipity() : LEDStripEffect(EFFECT_MATRIX_SERENDIPITY, "Serendipity")
+    PatternSerendipity() : LEDStripEffect(EFFECT_MATRIX_SERENDIPITY, "Serendipiti")
```

Comment:
I am just guessing, but my guess would be that this will make the name just fit on the HUB75. I seem to remember that we now lose the right-most pixels of the "y".

Reply:
Yup, it fits with an I